alembic/versions/fa0c72abb159_add_old_repositories.py

```diff
@@ -0,0 +1,47 @@
+"""Add old_repositories
```

Comment:
Maybe merge migrations into a single one? Since it's a first PR it's a bit strange there's already 3 migrations